gigantic/parser/model.py

```diff
@@ -74,15 +84,13 @@ class Archetype(UIResource):  # [HeroArchetypeName RxHeroProvider]
 	lowstaminatriggerpercent = 25.0  # 25.0f
 	lowstaminarecoveredpercent = 30.0  # 30.0f
 
-	def __repr__(self):
-		return self.heroarchetypename
-
 
 class_section_map = {
```

Comment:
Sorry that got misplaced, it was meant to refer to the class_section_map definition.